cmd/local.go

```diff
@@ -0,0 +1,233 @@
+//go:build !exclude_frontend
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os/exec"
+	"runtime"
+	"strings"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/kubernetes"
+	"github.com/kiali/kiali/log"
+	"github.com/kiali/kiali/util/httputil"
+)
+
+func newLocalCmd(conf *config.Config) *cobra.Command {
+	var (
+		homeClusterContext       string
+		kubeConfig               = kubernetes.KubeConfigDir()
+		remoteClusterContexts    []string
+		withoutBrowser           = false
+		portForwardToPromFlag    = false
+		portForwardToGrafanaFlag = false
+		portForwardToTracingFlag = false
+		enableTracing            = false
+		clusterNameOverrides     []string
+	)
```

Comment:
Interesting, I guess this introduces a new way for us to do CLI cmds, using this cobra package.  Maybe alphabeltize the vars.